purchase_order_generator_analytic/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Purchase Order Generator - Analytic
+===================================
+
+This module adds an analytic field to the purchase order generator.
+
+Installation
+============
+
+To install this module, you just need to select the module and make sure
+dependencies are available.
+
+Configuration
+=============
+
+No particular configuration to use this module.
+
+Usage
+=====
+
+To use this module, you need to:
+
+- Go on the configurator interface
+- Define the interval of time in which you would like to receive your products
+(ex: each week)
+- Choose the products and the quantities for each element of the sequence
+
+To generate purchase order lines thanks to the configurator you created:
+
+- Use the wizard
+- Choose the model you just created
+- Define a starting date, a partner, a destination location, an analytic
+account and an initial quantity
+- Validate to generate the purchase order lines
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
```

Comment:
can you add the images and the functional description here or in the doc/index.rst? (you decide) I think it is goos and helps for future usage.